### PR TITLE
docker: drop RPC flag and add REST API flags

### PR DESCRIPTION
**Pull request by jakubgs**
Related to removal of RPC API support in:
* https://github.com/waku-org/nwaku/pull/2416

@alrevuelta

### CODE REVIEW

File: templates/api.sh.j2
Comment (apentori) [inline templates/api.sh.j2:1]: Nice

**Review by apentori — APPROVED** [review]
LGTM

**Review by yakimant — APPROVED** [review]
Cool! Thanks for `api.sh`!

**Comment by alrevuelta** [comment]
Anything left here? We plan to merge https://github.com/waku-org/nwaku/pull/2416 soon. Unsure if you use releases or latest commits. But for sure that PR will be shipped with nwaku release v0.26.0.

Can we merge?

**Pull request merged**